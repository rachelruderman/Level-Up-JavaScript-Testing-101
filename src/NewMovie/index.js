import React, { Component } from 'react'
import MovieForm from './index'

export default class NewMovie extends Component {

    render() {
        return (
            <h1>
                New Movie
            </h1>
        )
    }
}
