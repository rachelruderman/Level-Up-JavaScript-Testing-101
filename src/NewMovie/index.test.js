import React from 'react';
import { render, fireEvent } from 'react-testing-library';
import NewMovie from './index';

test('<NewMovie/>', () => {
    const { debug, getByTestId, container } = render(<NewMovie/>);
    debug();
    expect(container.firstChild).toMatchSnapshot();
})