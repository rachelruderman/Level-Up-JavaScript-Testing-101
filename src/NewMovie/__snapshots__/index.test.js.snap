// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewMovie/> 1`] = `
<h1>
  New Movie
</h1>
`;
